test/daypickerinput/rendering.js

```diff
@@ -119,7 +119,9 @@ describe('DayPickerInput', () => {
       wrapper.instance().showDayPicker();
       wrapper.update();
       expect(wrapper.find('.DayPicker-Day--selected')).toHaveLength(1);
-      expect(wrapper.find('.DayPicker-Day--selected')).toHaveText('15');
+      expect(wrapper.find('.DayPicker-Day--selected')).toHaveText(
+        'Fri Dec 15 201715'
```

Comment:
This change is unexpected :)

Reply:
unfortunately, no matter how hard we hide it, the text the screenreader reads is still present. if they were two separate spans, we could select into them and verify each's content. could also use `toIncludeText` instead but that feels flakier :^(

Reply:
The tests are there to not break them... if they break, then is a breaking change, and I cannot work on a major release right now :(